x-pack/plugins/fleet/server/services/elastic_agent_manifest.ts

```diff
@@ -326,7 +328,7 @@ spec:
             # Elasticsearch API key used to enroll Elastic Agents in Fleet (https://www.elastic.co/guide/en/fleet/current/fleet-enrollment-tokens.html#fleet-enrollment-tokens)
             # If FLEET_ENROLLMENT_TOKEN is empty then KIBANA_HOST, KIBANA_FLEET_USERNAME, KIBANA_FLEET_PASSWORD are needed
             - name: FLEET_ENROLLMENT_TOKEN
-              value: "token-id"
+              value: ""
```

Comment:
Looks like it's needed as we replace this by the ENROLLMENT token in the instructions
```suggestion
              value: "token-id"
```

Reply:
oh nice let me change this

Reply:
https://github.com/elastic/kibana/blob/56db72f9b6b6b59ffe19699a5e0817e567ed06a2/x-pack/plugins/fleet/server/services/elastic_agent_manifest.ts 

Added in this PR https://github.com/elastic/kibana/pull/138393

Will close this PR